src/firstboot/firstboot.c

```diff
@@ -3,6 +3,7 @@
 #include <fcntl.h>
 #include <getopt.h>
 #include <linux/loop.h>
+#include <sys/file.h>
```

Comment:
Here also. Maybe all other places below?

Reply:
This file does have `lock-util.h` included, so yes, it can be removed here.